from PyQt5 import QtCore, QtGui, QtWidgets
from PyQt5.QtWidgets import QWidget


class Ui_MainWindow(QtWidgets.QMainWindow):
    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.resize(273, 464)
        font = QtGui.QFont()
        font.setPointSize(13)
        MainWindow.setFont(font)
        self.centralwidget = QtWidgets.QWidget(MainWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.label = QtWidgets.QLabel(self.centralwidget)
        self.label.setGeometry(QtCore.QRect(10, 10, 311, 31))
        font = QtGui.QFont()
        font.setPointSize(13)
        self.label.setFont(font)
        self.label.setObjectName("label")
        self.lineEdit_FullName_patient = QtWidgets.QLineEdit(self.centralwidget)
        self.lineEdit_FullName_patient.setGeometry(QtCore.QRect(10, 40, 251, 31))
        self.lineEdit_FullName_patient.setObjectName("lineEdit_FullName_patient")
        self.label_2 = QtWidgets.QLabel(self.centralwidget)
        self.label_2.setGeometry(QtCore.QRect(10, 80, 291, 31))
        font = QtGui.QFont()
        font.setPointSize(13)
        self.label_2.setFont(font)
        self.label_2.setObjectName("label_2")
        self.dateEdit_dateArrive_patient = QtWidgets.QDateEdit(self.centralwidget)
        self.dateEdit_dateArrive_patient.setGeometry(QtCore.QRect(10, 120, 251, 41))
        self.dateEdit_dateArrive_patient.setObjectName("dateEdit_dateArrive_patient")
        self.label_3 = QtWidgets.QLabel(self.centralwidget)
        self.label_3.setGeometry(QtCore.QRect(10, 170, 251, 31))
        font = QtGui.QFont()
        font.setPointSize(13)
        self.label_3.setFont(font)
        self.label_3.setObjectName("label_3")
        self.dateEdit_dateDisharge_patient = QtWidgets.QDateEdit(self.centralwidget)
        self.dateEdit_dateDisharge_patient.setGeometry(QtCore.QRect(10, 210, 251, 41))
        self.dateEdit_dateDisharge_patient.setObjectName("dateEdit_dateDisharge_patient")
        self.label_4 = QtWidgets.QLabel(self.centralwidget)
        self.label_4.setGeometry(QtCore.QRect(8, 260, 241, 31))
        self.label_4.setObjectName("label_4")
        self.lineEdit_diagnosis_patient = QtWidgets.QLineEdit(self.centralwidget)
        self.lineEdit_diagnosis_patient.setGeometry(QtCore.QRect(10, 300, 251, 31))
        self.lineEdit_diagnosis_patient.setObjectName("lineEdit_diagnosis_patient")
        self.pushButton_add_patient = QtWidgets.QPushButton(self.centralwidget)
        self.pushButton_add_patient.setGeometry(QtCore.QRect(69, 350, 131, 41))
        self.pushButton_add_patient.setObjectName("pushButton_add_patient")
        self.pushButton_exit_patient = QtWidgets.QPushButton(self.centralwidget)
        self.pushButton_exit_patient.setGeometry(QtCore.QRect(70, 400, 131, 41))
        self.pushButton_exit_patient.setObjectName("pushButton_exit_patient")
        MainWindow.setCentralWidget(self.centralwidget)

        self.retranslateUi(MainWindow)
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        _translate = QtCore.QCoreApplication.translate
        MainWindow.setWindowTitle(_translate("MainWindow", "MainWindow"))
        self.label.setText(_translate("MainWindow", "ФИО больного"))
        self.label_2.setText(_translate("MainWindow", "Дата поступления"))
        self.label_3.setText(_translate("MainWindow", "Дата выписки"))
        self.label_4.setText(_translate("MainWindow", "Диагноз"))
        self.pushButton_add_patient.setText(_translate("MainWindow", "Добавить"))
        self.pushButton_exit_patient.setText(_translate("MainWindow", "Отмена"))
